src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TaskStatus } from '../../types/Sort';
+
+interface FilterProps {
+  todos: Todo[];
+  sortType: TaskStatus;
+  onChangeSortType: (sortType: TaskStatus) => void;
+}
+
+export const Filter: FC<FilterProps> = ({
+  todos,
+  sortType,
+  onChangeSortType,
+}) => {
+  const active = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${active.length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
```

Comment:
check how you can reduce your code with array and `.map`

you have a lot of similar code with small differences